recipe/build-mlflow.sh

```diff
@@ -10,7 +10,7 @@ if [[ "${PKG_NAME}" == "mlflow-skinny" ]]; then
   export MLFLOW_SKINNY=1
   # https://github.com/mlflow/mlflow/pull/4134
   cp ${RECIPE_DIR}/README_SKINNY.rst ${SRC_DIR}
-  cp pyproject.skinny.toml pyproject.toml
+  cp skinny/pyproject.toml pyproject.toml
```

Comment:
I think we can remove this block and update line 25 to run `pip install skinny`.